compactHeadersApi.js

```diff
@@ -190,6 +190,67 @@ function install(window) {
     msgHeaderView.append(compactHeadersPopup);
   }
 
+  function patchRecipientClass() {
+    window.customElements.whenDefined("header-recipient").then(classHeaderRecipient => {
+      if (!classHeaderRecipient.prototype.originalUpdateRecipient) {
+        classHeaderRecipient.prototype.originalUpdateRecipient = classHeaderRecipient.prototype.updateRecipient;
+        classHeaderRecipient.prototype.updateRecipient = function() {
+          this.originalUpdateRecipient();
+
+          if (this.dataset.headerName == "to" || this.dataset.headerName == "cc") {
+            this.nameLine.textContent = this.displayName;
+            this.addressLine.textContent = this.emailAddress;
+            if (this.displayName) {
+              this.classList.add("has-display-name");
+            } else {
+              this.classList.remove("has-display-name");
+            }
+          }
+        }
```

Comment:
`patchRecipientClass` detours original class method `HeaderRecipient.updateRecipient` to:
1. Add name and email information to `To`/`CC` multiline element. 
   This information is only set for `From` header in thunderbird source code, we also add them for `To`/`CC` header now.
2. Add a class `has-display-name` for `To`/`CC` header, so whether they will show as multiline can be controlled by CSS.